bigquery/cloud-client/export_data_to_gcs.py

```diff
@@ -31,35 +31,33 @@
 from google.cloud import bigquery
 
 
-def export_data_to_gcs(dataset_name, table_name, destination):
+def export_data_to_gcs(dataset_id, table_id, destination):
     bigquery_client = bigquery.Client()
-    dataset = bigquery_client.dataset(dataset_name)
-    table = dataset.table(table_name)
-    job_name = str(uuid.uuid4())
+    dataset_ref = bigquery_client.dataset(dataset_id)
+    table_ref = dataset_ref.table(table_id)
+    job_id = str(uuid.uuid4())
```

Comment:
`job_id` is optional now that the client library can generate it, so probably can be removed from this sample